### DISCUSSION TITLE
[FUI-LESS] No install scripts or build support

### DISCUSSION THREAD

**Discussion by DeeeeLAN** [fomantic/Fomantic-UI#1719] [General] [answered]
Using the latest distribution of fomantic-ui-less (2.8.7). I noticed it doesn't kick off any install script that sets up a semantic.json file, nor does it create a default directory of less files. I copied the less directory and semantic.json file from a previous instance and running `npx gulp build` predictably did nothing. Is there a guide anywhere on using the theme-able FUI-LESS package?

**Comment by lubber-de** [comment, marked as answer]
That's not the intention of the LESS only Repo. The LESS repo is only supposed to be a lightweight themable version were you simply import the `semantic.less` file into your existing project which already compiles runs LESS elsewhere.
The first info inside that file shows

> Import this file into your LESS project to use Fomantic-UI without build tools

The full package including build scripts is this repo here only

It's also stated in the docs https://fomantic-ui.com/introduction/advanced-usage.html#less-only-distribution
